politeiawww/middleware.go

```diff
@@ -137,3 +139,57 @@ func recoverMiddleware(next http.Handler) http.Handler {
 		next.ServeHTTP(w, r)
 	})
 }
+
+func (p *politeiawww) getPassThrough(route string, r *http.Request) ([]byte, error) {
```

Comment:
Function needs a comment explaining what is included in the route string.  Is is `https://proposals.decred.org/api/v1/someRoute`, `/api/v1/someRoute`, or just `/someRoute`?  Same for the other function.  You shouldn't have to dig through a function to figure out how to use it.